client/src/actions/sftp.js

```diff
@@ -122,4 +122,25 @@ export const sftp_rm = (fm, session_id, cwd, files) => {
             fm.showAlert('Error: ' + error.message);
         }
     });
-};
\ No newline at end of file
+};
+
+export const sftp_chmod = (fm, cwd, name, mode, recursive) => {
+    fm.setState({
+        loading: true
+    })
+    axios.patch(`/sftp_chmod/${fm.session_id}`, {
+        path: `${cwd}/${name}`,
+        mode: mode,
+        recursive: recursive
+    }).then(_ => {
+        fm.loadDir(cwd)
```

Comment:
previously it was:
window.location.reload()

which does reload the FM. 

However,
it will not work if the cwd is ~/Downloads/some/very/deep/directory : the cwd will default back to ~/ , which might lead to a bad experience to the user